src/main/java/com/baidu/hugegraph/backend/store/tikv/TikvStdSessions.java

```diff
@@ -377,11 +377,13 @@ public void put(String table, byte[] key, byte[] value) {
 
         @Override
         public void put(String table, byte[] key, byte[] value, long ttl) {
-            Map<ByteString, ByteString> batch =
-                            this.putBatchTTL.getOrDefault(ttl,
-                                 new HashMap<ByteString, ByteString>());
+            Map<ByteString, ByteString> batch = this.putBatchTTL.get(ttl);
+            if (batch == null) {
+                batch = new HashMap<>();
+                this.putBatchTTL.putIfAbsent(ttl, batch);
```

Comment:
consider the put fails:
```java
oldBatch = this.putBatchTTL.putIfAbsent(ttl, batch);
if (oldBatch != null) {
    batch = oldBatch
}
```

Reply:
The caller should be the same thread, so there failure is unexpected;

Reply:
can use map.put() if won't be called by multi threads

Reply:
ok, can we use this code?
  Map<ByteString, ByteString> batch =
                this.putBatchTTL.computeIfAbsent(ttl, __ -> {
                    return new HashMap<>();
                });